src/components/Ecosystem/styles.module.css

```
@@ -0,0 +1,240 @@
+}
+
+.header p {
+  margin: 0 auto 22px;
+  color: #5f6b7a;
+  font-size: 0.9rem;
+  line-height: 1.65;
+}
+
+.button {
+  margin-top: 18px;
+  display: inline-flex;
```

Stylelint (17.11.1)

[error] Duplicate property "display" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)